chocolatey/datadog-agent.nuspec

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>$package_id$</id>
+    <!-- Will be replaced during packaging -->
+    <version>$package_version$</version>
+    <packageSourceUrl>https://github.com/DataDog/datadog-agent/tree/master/chocolatey</packageSourceUrl>
+    <owners>package@datadoghq.com</owners>
+    <title>Datadog Agent</title>
+    <authors>Datadog</authors>
+    <projectUrl>https://github.com/DataDog/datadog-agent</projectUrl>
+    <iconUrl>https://datadog-prod.imgix.net/img/dd_logo_70x75.png</iconUrl>
+    <copyright>$copyright$</copyright>
+    <licenseUrl>https://raw.githubusercontent.com/DataDog/datadog-agent/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/DataDog/datadog-agent</projectSourceUrl>
+    <docsUrl>https://docs.datadoghq.com</docsUrl>
+    <tags>datadog agent monitoring admin</tags>
+    <summary>The Datadog Agent for Microsoft Windows</summary>
+    <description>The Datadog Agent faithfully collects events and metrics and brings them to Datadog on your behalf so that you can do something useful with your monitoring and performance data.</description>
+    <releaseNotes>$release_notes$</releaseNotes>
```

Comment:
Maybe we can link the CHANGELOG file here. Otherwise, I think this will be empty (or will `$release_notes$` be replaced somehow with the actual release notes?)

Reply:
Nevermind, just saw what fills this field.